### ISSUE TITLE
5 sampling dates - can we put all 5 depth profiles on a page?

### ISSUE THREAD

**Issue by sbmarsh-USGS** [USGS-R/lakeReport#11]
Many of the lakes have 5 sampling dates.  Can we get all 5 profile dates on one page?

- [ ] 432409088151600 (Big Cedar Lake North)

- [ ] 432244088154900 (Big Cedar Lake South)

- [ ] 423329088323300 (Geneva Lake West End)

- [ ] 430551088273500 (Oconomowoc Center)

- [ ] 430609088262200 (Oconomowoc Hewitt Point)

- [ ] 423246088175800 (Powers Lake)

- [ ] 424915088083900 (Wind Lake)

**Comment by lindsaycarr** [comment]
for 5 on a page, need to change the figure height from 6.5 to 8. 

```r
opts_chunk$set(
  fig.height = 8
)
```
Also, need to edit these lines from 4 to 5 plots:
```r
num_2x4_plots <- floor(length(dates_uniq)/5)
num_extra_dates <- length(dates_uniq)%%5

dts <- dates_uniq[d:(p*5)]
  
plot_setup <- matrix(1:10, 2, 5)
```

**Comment by lindsaycarr** [comment]
@sbmarsh-USGS merged in a fix for this with #36. Please test and close if it's working!

**Comment by sbmarsh-USGS** [comment]
Awesome! Thank you!
